packages/cfd/src/Helpers/constants.ts

```diff
@@ -36,10 +36,8 @@ const getTitle = (market_type: string, is_eu_user: boolean) => {
 const REAL_DXTRADE_URL = 'https://dx.deriv.com';
 const DEMO_DXTRADE_URL = 'https://dx-demo.deriv.com';
 
-const CTRADER_DESKTOP_DOWNLOAD = 'https://getctrader.com/deriv/ctrader-deriv-setup.exe';
-
+const CTRADER_DESKTOP_DOWNLOAD = 'https://deriv.ctrader.com/cbrokerdemo-deriv-setup.exe';
```

Comment:
This URL is wrong it is of cBroker platform not cTrader please revert these URL chnages
```suggestion
const CTRADER_DESKTOP_DOWNLOAD = 'https://getctrader.com/deriv/ctrader-deriv-setup.exe';
```

Reply:
from where did you get this link?

Reply:
You can refer to production
<img width="890" alt="Screenshot 2023-09-21 at 5 26 43 PM" src="https://github.com/binary-com/deriv-app/assets/120543468/f7f59ca2-46c5-466d-8937-c2b4510a774f">